extensions/notion/src/components/forms/CreatePageForm.tsx

```diff
@@ -77,8 +77,8 @@ export function CreatePageForm({ mutate, launchContext, defaults }: CreatePageFo
 
   const databasePropertyIds = databaseProperties.map((dp) => dp.id) || [];
 
-  const initialValues: Partial<CreatePageFormValues> = { database: databaseId ?? undefined };
-  const validation: Parameters<typeof useForm<CreatePageFormValues>>[0]["validation"] = {};
+  const initialValues: Partial<CreateDataBasePageFormValues> = { database: databaseId ?? undefined };
```

Comment:
Hi there, great job! I use the Notion extension, which is why I'm checking this PR.
`databaseId ?? undefined` Does it really make sense?
I guess you don't need to type the database state with null and it will work
![Image 29 05 24 at 10 23](https://github.com/raycast/extensions/assets/108681764/f44797d3-d2dd-44d8-abf6-6305fc8a7e83)

Reply:
> Hi there, great job! I use the Notion extension, which is why I'm checking this PR. `databaseId ?? undefined` Does it really make sense? I guess you don't need to type the database state with null and it will work ![Image 29 05 24 at 10 23](https://private-user-images.githubusercontent.com/108681764/334700824-f44797d3-d2dd-44d8-abf6-6305fc8a7e83.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NjkwMzAsIm5iZiI6MTcxNjk2ODczMCwicGF0aCI6Ii8xMDg2ODE3NjQvMzM0NzAwODI0LWY0NDc5N2QzLWQyZGQtNDRkOC1hYmY2LTYzMDVmYzhhN2U4My5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQwNzQ1MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODdkMGUwZTFhY2UxM2ZhOWU1MDg3NjI3NGM4ZTUxMDk4Y2I2NmYxYTI3MDBiNGI3NGNlNjIxNmRiOGNmY2VlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2xnFCvTLGcDH4Sz2jVIrVBBnnuEZnR1T934hRR_EMMQ)

Hi, I appreciate your comment.
This code please @bkeys818 help answer, I find you first commit this logic.